.github/workflows/build-bluefin-toolbox.yml

```
@@ -24,6 +24,8 @@ jobs:
       id-token: write
     strategy:
       fail-fast: false
+      matrix:
+        tag: ["latest", "latest-dev"]
```

```suggestion
        tag: ["latest", "testing"]
```

Let's copy bazzite here so we remain consistent across the org?

---

Sounds good to me :)